assets/src/components/BarChart.js

```diff
@@ -1,11 +1,17 @@
+import React, { useState } from 'react'
 import createBarChart from './d3/createBarChart'
-import withResponsiveness from './withResponsiveness'
-import createChartComponent from './createChartComponent'
-import compose from '../util/compose'
-
-const BarChart = compose(
-  withResponsiveness,
-  createChartComponent
-)(createBarChart)
+import useResponsiveness from '../hooks/useResponsiveness'
+import useCreateChart from '../hooks/useCreateChart'
```

Comment:
This can be deleted as well as we are not using it either for Assignment view or files view

Reply:
Okay I've removed this one as well

Reply:
I've deleted these now